lib/constants/spacing.dart

```diff
@@ -76,3 +76,7 @@ const popupTitleGap = Gap(40);
 /// Gap before the bottom divider in the display pages.
 
 const textPageBottomGap = Gap(20);
+
+/// Width to fit 5 ChoiceChips in a Row.
+
```

Comment:
If a constant is only used in one location then define it there.

See https://survivor.togaware.com/gnulinux/flutter-style-constants.html